### PR TITLE
Add rel="noopener" to external links

### PR DESCRIPTION
**Pull request by yuku-t**
When we added the rel="nofollow" and target="_blank" to the external links, some experts were wondering whether we would also think about adding rel="noopener" as well.

http://b.hatena.ne.jp/entry/blog.qiita.com/post/149486954709/externallinkattrib

rel="noopener" is described in the following blogs.

- https://mathiasbynens.github.io/rel-noopener/
- https://blog.jxck.io/entries/2016-06-12/noopener.html
- https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/

In short, it prevents users from being affected with some security attack.

### CODE REVIEW

**Comment by yuku-t** [comment]
@yoichiro review please 🙏

**Comment by yoichiro** [comment]
@yuku-t LGTM. Thanks. 👍
